frappe/utils/boilerplate.py

```diff
@@ -298,14 +298,13 @@ def _create_parent_folder_if_not_exists(self):
 
 """
 
-hooks_template = """from . import __version__ as app_version
```

Comment:
It's used to render versions on desk. 

Maybe add `# noqa` to avoid linting messages (in template and apps)